ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_elementwise.py

```diff
@@ -822,3 +822,50 @@ def test_digamma(
         on_device=on_device,
         x=x[0],
     )
+
+
+@st.composite
+def _sparsify_tensor_stg(draw):
+    dtype, tensor, shape = draw(
+        helpers.dtype_and_values(
+            available_dtypes=helpers.get_dtypes("numeric"),
+            ret_shape=True,
+            min_num_dims=1,
+            min_dim_size=10,
+            min_value=10,
+        )
+    )
+
+    size = 1
+    for dim in shape:
+        size *= dim
+
+    card = draw(st.integers(min_value=2, max_value=size - 1))
+
+    return dtype, tensor[0], card
+
+
+# sparsify_tensor
+@handle_test(
+    fn_tree="functional.ivy.experimental.sparsify_tensor",
+    tensor_data=_sparsify_tensor_stg(),
+    test_with_out=st.just(False),
```

Comment:
```suggestion
```
The function does have an `out` argument, so, we should be testing for it as well :)